### PR TITLE
↕️ Increment input button gas limit by 1000

### PR DESCRIPTION
**Pull request by henryboldi**
This PR adds incrementing by one thousand on the gas limit field. Incrementing by one is too fine-grained in this context, so here we've increased it!⛽️

https://github.com/tallycash/extension/issues/1369

### CODE REVIEW

**Comment by 0xDaedalus** [comment]
Doesn't seem to be working for me:

https://user-images.githubusercontent.com/94649004/168306521-ae9f7a99-5112-4a27-beb2-04673c4f4519.mov

**Comment by henryboldi** [comment]
> Doesn't seem to be working for me:

I believe that's the deprecated version. I branched off of `gas-gas` and this edit is only on the new gas page 😬

**Comment by 0xDaedalus** [comment]
ah - can we change the deprecated version as well please?  since we don't know when we'll be turning off the feature flag.

**Comment by henryboldi** [comment]
Absolutely! 🙏

File: ui/components/Shared/SharedInput.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ interface Props<T> {
   parseAndValidate: (
     value: string
   ) => { parsed: T | undefined } | { error: string }
+  step?: number
```
[the remark this answers is not in the document]
Comment (henryboldi) [reply ui/components/Shared/SharedInput.tsx:20]: Very true, this did feel a bit icky 😬

File: ui/components/Shared/SharedInput.tsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ export function SharedTypedInput<T = string>(props: Props<T>): ReactElement {
     onFocus,
     value: currentValue,
     errorMessage,
+    step = 1,
```
[the remark this answers is not in the document]
Comment (henryboldi) [reply ui/components/Shared/SharedInput.tsx]: Roger that! ✔️ Thank you

**Review by 0xDaedalus — APPROVED** [review]
LGTM 👍

**Pull request merged**